src/plugins/media-analysis/media-analysis.yml

```diff
@@ -25,3 +25,6 @@ importFromCsv:
 segmentsDropzoneInfo:
   en: You can drop a CSV file in this box in order to import segments
   de: Sie können eine CSV-Datei in diese Box ziehen, um Segmente zu importieren
+startPlaybackFromHere:
+  en: Click here to start the playback beginning with this part
```

Comment:
How about "Click here to play from this part"? I find it might be clearer, at least in English 🤔 
For whoever does read it, I mean 😅

Reply:
Maybe someone will read it :sweat_smile:

I had something along these lines initially but it's hard to translate that to German nicely. I changed it to your suggestion (but only the English text).